### PR TITLE
Improve the compression project settings documentation

### PR DESCRIPTION
**Pull request by Calinou**

### CODE REVIEW

File: doc/classes/ProjectSettings.xml [hunk 1]
```diff
@@ -260,19 +260,19 @@
...
-			Default compression level for Zlib. Affects compressed scenes and resources.
+			The default compression level for Zlib. Affects compressed scenes and resources. Higher levels result in smaller files at the cost of compression speed. Decompression speed is mostly unaffected by the compression level. [code]-1[/code] uses the default gzip compression level, which is identical to [code]6[/code] but could change in the future due to underlying zlib updates.
 		</member>
 		<member name="compression/formats/zstd/compression_level" type="int" setter="" getter="" default="3">
-			Default compression level for Zstandard. Affects compressed scenes and resources.
+			The default compression level for Zstandard. Affects compressed scenes and resources. Higher levels result in smaller files at the cost of compression speed. Decompression speed is mostly unaffected by the compression level.
 		</member>
 		<member name="compression/formats/zstd/long_distance_matching" type="bool" setter="" getter="" default="false">
-			Enables long-distance matching in Zstandard.
+			Enables [url=https://github.com/facebook/zstd/releases/tag/v1.3.2]long-distance matching[/url] in Zstandard.
```
Comment (Calinou) [inline doc/classes/ProjectSettings.xml:272]: This was the best documentation I could find about it from a quick search. Feel free to suggest a more "persistent" resource.

Comment (akien-mga) [reply doc/classes/ProjectSettings.xml:272]: Found https://facebook.github.io/zstd/zstd_manual.html but there's no way to directly reference the relevant part. Last time I checked the 1.3.2 release notes were indeed the best reference I could find, beyond the source code.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.2.

**Pull request merged**